shared/js/trackers.js

```diff
@@ -46,7 +46,25 @@ require.scopes.trackers = (function() {
         if (settings.getSetting('trackerBlockingEnabled')) {
 
             let parsedUrl = tldjs.parse(urlToCheck)
-            let urlSplit = parsedUrl.hostname.split('.')
+            let hostname
+
+            if (parsedUrl && parsedUrl.hostname) {
+                hostname = parsedUrl.hostname
```

Comment:
@andrey-p Hey, not sure if you saw it, but we have a [util function to get the hostname](https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/develop/shared/js/utils.js#L6) - is that something that could have been useful here?

Reply:
Ah, didn't realize that existed. It uses `tldjs` under the hood. Maybe we can consolidate the two.